bindings/python/dlite-collection.i

```diff
@@ -97,8 +142,8 @@ struct _DLiteCollection {
   struct _CollectionIter {
     DLiteCollection *coll;
     DLiteCollectionState state;
-    char *s, *p, *o;  /* search pattern */
-    char rettype;     /* return type: I=Instance, R=Relation, T=tuple */
+    char *s, *p, *o, *d;  /* search pattern */
+    char rettype;         /* return type: I,R,T,t,s,p,o,d */
```

Comment:
Whye do you not have I=Instance, R=Relation, T=tuple, t=tuple.... etc here as before? (much easier for me to read and know what we are considering)

Reply:
It guess that I was lazy. I updated the comment now.